aiapy/response/tests/test_channel.py

```diff
@@ -114,18 +114,18 @@ def test_effective_area(channel) -> None:
     ("source", "eve_correction_truth"),
     [
         pytest.param(
-            9,
-            0.9494731307817633 * u.dimensionless_unscaled,
+            "SSW",
+            0.95484152 * u.dimensionless_unscaled,
```

Comment:
Do we know why these values changed?

Reply:
No.

Reply:
I guess in the SSW case, since its older and not updated.